# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: media/mtransport/ipc/ProxyConfigLookup.h
# Commit: 6e1eba5c88ad
# Full Hash: 6e1eba5c88ad47f0a2a3aa37b4c2cae9f31c2ef3
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1569183
# File Overlap Count: 1
# Description:
#   Bug 1569183: Stop doing a proxy lookup to determine whether we're configured to use a proxy (for the proxy_only_if_behind_proxy pref), and instead look at whether we loaded the doc using a proxy. r=mjf,mayhemer,jld
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D45289
# ==============================================================================

diff -r c43589fd0f90 -r 6e1eba5c88ad media/mtransport/ipc/ProxyConfigLookup.h
--- a/media/mtransport/ipc/ProxyConfigLookup.h	Tue Sep 17 18:18:33 2019 +0000
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,33 +0,0 @@
-/* -*- Mode: C++; tab-width: 8; indent-tabs-mode: nil; c-basic-offset: 2 -*- */
-/* vim: set ts=8 sts=2 et sw=2 tw=80: */
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-#ifndef mozilla_net_ProxyConfigLookup_h
-#define mozilla_net_ProxyConfigLookup_h
-
-#include "nsIProtocolProxyCallback.h"
-#include <functional>
-
-namespace mozilla {
-namespace net {
-
-class ProxyConfigLookup final : public nsIProtocolProxyCallback {
- public:
-  NS_DECL_ISUPPORTS
-  NS_DECL_NSIPROTOCOLPROXYCALLBACK
-
-  static nsresult Create(std::function<void(bool)>&& aCallback);
-
- private:
-  explicit ProxyConfigLookup();
-  ~ProxyConfigLookup();
-
-  std::function<void(bool)> mCallback;
-};
-
-}  // namespace net
-}  // namespace mozilla
-
-#endif  // mozilla_net_ProxyConfigLookup_h